examples/nodejs/src/helpers.ts

```diff
@@ -29,9 +29,11 @@ export async function fetchRepo(name: string, subgraph: string) {
   const wrapper = new GraphQLWrapper(subgraph)
 
   // Invoke the custom query and receive data
-  return wrapper.performQuery(QUERY_REPO_BY_NAME, {
+  const { data } = await wrapper.performQuery(QUERY_REPO_BY_NAME, {
     name,
   })
+
+  return data.repos[0]
```

Comment:
Hmmm, if this returns null (due to a network error or something of the sort), this may just explode. This is probably the reason that makes connectors break if we update our subgraphs as well. Should we prefix it with `?`, or should we just let the user handle it?

Reply:
(this was part of another PR sorry :s)

It seems like it yes. We should definitely improve the reliability of the parts where data is fetched, it’s something I want to do soon alongside providing better errors. I also think we should consider `GraphQLWrapper` to be an object used internally or by app connectors, but not by end users or demos.